drivers/overlay/overlay.go

```diff
@@ -2532,6 +2532,20 @@ func nameWithSuffix(name string, number int) string {
 	return fmt.Sprintf("%s%d", name, number)
 }
 
+func validateOneAdditionalLayerPath(target string) error {
+	for _, p := range []string{
+		filepath.Join(target, "diff"),
+		filepath.Join(target, "info"),
+		filepath.Join(target, "blob"),
+	} {
+		if err := fileutils.Exists(p); err != nil {
+			wrapped := fmt.Errorf("failed to stat additional layer %q: %w", p, err)
+			return fmt.Errorf("Additional Layer Store %v: %w", target, wrapped)
```

Comment:
AFAICS the log entry will now be something like `Additional Layer Store /var/foo/docker.io/library/sha256:…: failed to stat additional layer /var/foo/docker.io/library/sha256:…/diff: faccessat /var/foo/docker.io/library/sha256:…/diff: no such file or directory`

That still seems excessive. Also the `wrapped` step is unnecessary, use one `fmt.Errorf` to create the whole message.